src/pages/HealthContent.tsx

```diff
@@ -1,10 +1,133 @@
 import React from "react";
+// @ts-ignore
+import workoutImg from "../assests/images/workout.jpg";
 import styles from "./HealthContent.module.css";
+import CardPage from "../UI/CardPage";
+import Carousel from "react-multi-carousel";
+import "react-multi-carousel/lib/styles.css";
 
 const HealthContent = () => {
+  const cards = [
```

Comment:
this data could be moved out of the component.  I believe you said you were going to work on pulling the data in on an ajax request though, right?

Reply:
correct